### ISSUE TITLE
Getting Attention Activations to Visualize Attention in Seq2Seq

### ISSUE THREAD

**Issue by nicholaslocascio** [tensorflow/tensorflow#1668]
All attention papers feature some visualization of the attention weights on some input. Has anyone been able to run a sample through the Seq2Seq Attention Decoder model in `translate.py` and get the attention activations to do such a visualization?

**Comment by keveman** [comment]
The attention mask is available as a tensor here : 
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/ops/seq2seq.py#L522

It should be easy to fetch it out during a run call and visualize it. You can try posting this to StackOverflow  to see if someone in the general community has done this visualization. I am closing this issue, since we have the required functionality in TensorFlow.

**Comment by jarheadfa** [comment]
The link is broken. What is the correct link?

**Comment by guotong1988** [comment]
same problem
